tests/snapshots/tests.contrib.httpx.test_httpx.test_request_headers.snap

```diff
@@ -0,0 +1,21 @@
+[[{"name" "http.request"
+   "service" nil
+   "resource" "http.request"
+   "type" "http"
+   "error" 0
+   "span_id" 0
+   "trace_id" 0
+   "parent_id" nil
+   "start" 1620689782941177000
+   "duration" 8040000
+   "meta" {"runtime-id" "0975fd0faf2a4855b8e668f3a21af4d4"
+           "http.method" "GET"
+           "http.url" "http://localhost:8001/response-headers"
+           "http.status_code" "200"
+           "http.request.headers.some-request-header" "Request-Value"
```

Comment:
though we have a snapshot here we should have a way to ensure that these tags are being set from the test case. what do you think?

Reply:
do you have a concern with it being a snapshot test?

Reply:
My concern would be that these are useful for regression testing but we should have asserts in the test for the data that is expected. Otherwise, it's up for human inspection to ensure the first snapshot has all the correct data.

Reply:
makes sense to me

Reply:
well... I actually think keeping as snapshots make sense, otherwise we are basically just going to re-assert the same assertions we get from the snapshot tests.

the first time a snapshot is added it _should_ be reviewed by humans to be sure it is correct. once it gets merged it is set in stone, we should definitely know that it is doing the right thing